web/src/components/NavigationBarScrollContainer.tsx

```diff
@@ -61,8 +61,10 @@ const getScrollableWidth = (scrollContainer: RefObject<HTMLDivElement>): number
 
 const NavigationBarScrollContainer = ({ children, direction, scrollContainerRef }: PropsType): ReactElement => {
   const [scrollPosition, setScrollPosition] = useState(0)
+
+  const showArrows = scrollContainerRef.current ? getScrollableWidth(scrollContainerRef) > 0 : false
   const showArrowLeft = scrollContainerRef.current ? scrollPosition > 0 : false
-  const showArrows = scrollContainerRef.current ? scrollPosition < getScrollableWidth(scrollContainerRef) : false
+  const showArrowRight = scrollContainerRef.current ? scrollPosition < getScrollableWidth(scrollContainerRef) : false
```

Comment:
I would prefer to adjust the naming here. `showArrows` is a different `show` than `showArrowLeft` and `showArrowRight` (not rendering arrows vs just hiding the arrows). Maybe `scrollContainerNeeded` or something like that?

Reply:
you are right but i think it should be named like `showArrowContainer` because the scrollContainer is rendered in any case

Reply:
Works for me, just different :)